editor/editor_run.h

```diff
@@ -51,7 +51,7 @@ class EditorRun {
 
 public:
 	Status get_status() const;
-	Error run(const String &p_scene, const String &p_custom_args, const List<String> &p_breakpoints);
+	Error run(const String &p_scene, const String &p_custom_args, const List<String> &p_breakpoints, const bool &skip_breakpoints);
```

Comment:
How about give `skip_breakpoints`  as default value as  `bool p_skip_breakpoints = false`.
In this way it should not break anything.

Reply:
And is it breaking anything? It seems to be called only in one place.

Reply:
fixed anyway